www/FingerPrint.js

```diff
@@ -7,42 +7,27 @@ var FingerPrint = function () {};
 // method to check is the touch id option is available in the device
 FingerPrint.prototype.isAvailable = function (successCallback, errorCallback) {
   // success callback function for android 
-  var successCbFingerPrintAuth = function(result) {
-    if(result && !result.isAvailable){
-      var message;
-      // if not support touch id
-      if(!result.isHardwareDetected){
-        message = "This device didn't support Touch ID";
-      }
-      // if not have any finger print enrolled in the device
-      else if(!result.hasEnrolledFingerprints){
-        message = "No fingers are enrolled with Touch ID";
-      }
-      errorCallback(message);
+
+  function isAvailableSuccess(result) {
+    var biometricType = 3;
```

Comment:
what does it mean this 3 value?

Reply:
this is a value for an outsystems "None" record